test/translate_ligo_fn.coffee

```diff
@@ -297,15 +294,18 @@ describe "translate ligo section fn", ()->
         assert((sum >= test_reserved_long___self));
       } with (sum);
     
-    function test (const #{config.reserved}__unit : unit) : (list(operation) * nat) is
+    function test (const #{config.reserved}__unit : unit) : (nat) is
       block {
         const n : nat = abs(not (0));
-        exactAdd(n, 1n);
-      } with ((nil: list(operation)), 0n);
+        const terminate_tmp_0 : (nat) = exactAdd(n, 1n);
```

Comment:
What is terminate?

Reply:
There is device called terminator, which is used to terminate end of network cable (to improve signal quality).
In this case situation is similar. We need terminate expression to statement with var decl. Terminate is signalling that this variable is unused intentionally (opposint to regular tmp, that used for unpack tuples)